extensions/core-xml/deployment/src/test/resources/routes.1

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why not with an `.xml` suffix? - That would be more practical for editing and also for the license plugin.